### PR TITLE
Update fr.json

### PR DESCRIPTION
**Pull request by alex-quiniou**
Update many strings in french

### CODE REVIEW

File: src/assets/i18n/fr.json [hunk 1]
```diff
@@ -40,11 +40,11 @@
   "Applications": "",
   "Applied Dataset Quota": "",
   "Apply Owner": "",
-  "Are you sure you want to delete \"{cert}\"?": "",
-  "Are you sure?": "",
-  "Attention": "",
+  "Are you sure you want to delete \"{cert}\"?": "Êtes-vous sûr de vouloir supprimer "{cert}\"?",
```
Comment (undsoft) [inline src/assets/i18n/fr.json:43]: Should be:
```
"Êtes-vous sûr de vouloir supprimer \"{cert}\"?"
```

Comment (alex-quiniou) [reply src/assets/i18n/fr.json:43]: Oops my bad.
Do I have to change it myself ?

Comment (undsoft) [reply src/assets/i18n/fr.json:43]: Yes

**Review by undsoft — APPROVED** [review]
Thanks!

**Pull request merged**